spec/lib/util.spec.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+describe("util", () => {
+    var util;
+
+    beforeEach(() => {
+        util = require("../../lib/util")();
+    });
+    describe(".clone()", () => {
+        it("makes a deep copy", () => {
```

Comment:
Need another test which doesn't pass in an object. We are only testing one branch of utils.